codegen/templates/http_client.tmpl

```diff
@@ -98,22 +99,79 @@ func (c *{{$clientName}}) {{title .Name}}(
 		respHeaders[k] = res.Header.Get(k)
 	}
 
-	res.CheckOKResponse({{.OKStatusCode.Code}})
+	res.CheckOKResponse([]int{
```

Comment:
This method still logs a warning to disk where as the default block only returns an error.

I want to keep the warning logic here.